source/scripting_v3/GTA/Camera/Camera.cs

```diff
@@ -555,6 +555,20 @@ public void AttachTo(PedBone pedBone, Vector3 offset)
 			Function.Call(Hash.ATTACH_CAM_TO_PED_BONE, Handle, pedBone.Owner.Handle, pedBone, offset.X, offset.Y, offset.Z, true);
 		}
 
+		/// <summary>
+		/// Attaches this <see cref="Camera"/> to a specific bone on a specific <see cref="Vehicle"/>.
+		/// </summary>
+		/// <param name="vehicle">The vehicle to attach the camera to.</param>
+		/// <param name="boneId">The name of the vehicle bone to attach the camera to.</param>
+		/// <param name="offset">Offset relative to the specified <paramref name="boneId"/>.</param>
+		/// <param name="rotation">Rotation vector of the <see cref="Camera"/>.</param>
+		/// <param name="relativeRotation">Set <see langword="true" /> to have rotation relative to the specified <paramref name="boneId"/>; else set <see langword="false" />.</param>
+		/// <param name="fixedDirection">Set <see langword="true" /> to have camera direction match the <see cref="Vehicle"/> direction; else set <see langword="false" />.</param>
+		public void AttachTo(Vehicle vehicle, string boneId, Vector3 offset, Vector3 rotation, bool relativeRotation, bool fixedDirection)
```

Comment:
We could use 1 `EntityBone` param rather than 2 `Vecicle` and `string` params, so we can reduce the number of the params and let script devs retrieve a bone as they like to do (maybe some of them specifies by index).